core/tls.cabal

```diff
@@ -160,6 +160,7 @@ Benchmark bench-tls
                    , QuickCheck >= 2
                    , tasty-quickcheck
                    , tls
+  ghc-options:       -Wall -fno-warn-unused-imports -fno-warn-orphans
```

Comment:
If I would like to move the `no-warn-orphans` flag to files to explicitly indicate which files contain orphan instances.